src/hooks/use-observable.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { useState, useEffect, useMemo } from 'react'
+import { BehaviorSubject } from 'rxjs/BehaviorSubject'
+import type { Observable } from 'rxjs/Observable'
+
+export function useObservable(observable$: Observable, initialState, inputs): any {
```

Comment:
❤️  — can you submit this into Nozbe/withObservables repo? I think it only makes sense to add it there, as `withObservables` is not part of 🍉 either, but a more general component.